modules/gdscript/gdscript_parser.cpp

```diff
@@ -359,6 +364,8 @@ Error GDScriptParser::parse(const String &p_source_code, const String &p_script_
 	parse_program();
 	pop_multiline();
 
+	memdelete(text_tokenizer);
```

Comment:
```suggestion
	memdelete(text_tokenizer);
	tokenizer = nullptr;
```
To avoid dangling pointers